test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/CsdlReaderTests.cs

```diff
@@ -2161,6 +2166,85 @@ public void ImportSchemaElementsOnlyFromReferencedNamespacesl()
             Assert.Null(personType);
         }
 
+        // We currently do not support reading the scale and srid values if the value is variable or Variable 
+        // The IEdmDecimalTypeReference'Scale value should have the value `variable` or `Variable` depending
+        // On what is in the CSDL. 
+        [Fact]
+        public void ReadingLegacyScaleAndSridVariable_WithoutAnySetting_NotSupported()

```

Comment:
Combine this test with `ReadingLowerCaseScaleAndSridVariable_WithoutAnySetting_NotSupported` into a single parameterized test. Less code to maintain...